lib/core/stream.nit

```diff
@@ -417,6 +417,9 @@ abstract class Writer
 	# Write a single byte
 	fun write_byte(value: Byte) is abstract
 
+	# Writes a single char
+	fun write_char(c: Char) do write(c.to_s)
```

Comment:
I'm not fan of this one since `to_s` cause a expensive allocation thus might be not POLA. (Note that I have no better proposal)